mollie-payments-for-woocommerce/includes/mollie/wc/plugin.php

```diff
@@ -423,6 +412,27 @@ public static function addGateways( array $gateways ) {
 		return $gateways;
 	}
 
+    /**
+     * Disable Apple Pay Gateway
+     *
+     * @param array $gateways
+     * @return array
+     */
+    public static function maybeDisableApplePayGateway(array $gateways)
+    {
+        $gateways = array_merge($gateways, self::$GATEWAYS);
+        $postData = (string)filter_input(INPUT_POST, 'post_data', FILTER_SANITIZE_STRING) ?: '';
+
+        if ($postData) {
```

Comment:
I suggest inverting conditions that lead to an action not being taken. For example, I would rewrite this particular selection as

```
if (!$postData) {
  return $gateways;
}
```

This improves readability, and even makes the method easier to write, and to change later.